### PR TITLE
Update activity-feed.md and templates.md documentation

### PR DESCRIPTION
**Pull request by michizhou**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Update activity-feed.md and templates.md by fixing documentation

- **Why was this change needed?** (You can also link to an open issue here)

Changes were needed for improved readability and consistency

- Changed `organisational` to `organizational` in templates.md
- Fix grammatical errors and added or removed punctuation

- **Other information**:

### CODE REVIEW

**Comment by michizhou** [comment]
@p-fernandez @scopsy @davidsoderberg Please feel free to let me know if additional changes are needed.

File: docs/docs/platform/templates.md [hunk 1]
```diff
@@ -6,33 +6,33 @@ sidebar_position: 1
...
+Let's explore the different parts of a notification template.
 
 ## Global template metadata
 
 This will contain general information regarding the notification itself. Let's explore the different aspects of it.
 
 ### Notification name
 
-The name will be used to identify the notification template when triggering it. A slugified version of the name will be generated after the notification was created. For example, a notification template with the name of "Test Notification" will be converted to "test-notification" as the trigger key.
+The name used to identify the notification template when triggering it. A slugified version of the name will be generated after the notification was created. For example, a notification template with the name of "Test Notification" will be converted to "test-notification" as the trigger key.
```
Comment (p-fernandez) [inline docs/docs/platform/templates.md:17]: Please keep the existing copy.

File: docs/docs/platform/templates.md [hunk 2]
```diff
@@ -6,33 +6,33 @@ sidebar_position: 1
...
 
 ### Notification Group
 
-The notification group is used to group multiple notification templates into a single group, currently only used behind the scenes for organisational purposes. But, in the upcoming subscriber preferences management, it will be used to group multiple notifications for the subscriber.
+The notification group is used to group multiple notification templates into a single group (currently only used behind the scenes for organizational purposes). But, in the upcoming subscriber preferences management, it will be used to group multiple notifications for the subscriber.
```
Comment (p-fernandez) [inline docs/docs/platform/templates.md:21]: I am sorry but this copy doesn't provide improvements to be honest.
`organisational` is rightly spelled (british english). It is undecided an unique language variaton for the whole codebase.
@scopsy maybe something to agree and to add to the CONTRIBUTING.md file.

File: docs/docs/platform/templates.md [hunk 3]
```diff
@@ -6,33 +6,33 @@ sidebar_position: 1
...
 
-The templates steps are used to organize the different messages in a particular flow. You can add messages for multiple channels and in the upcoming workflow editor release you will be able to configure custom flows that includes actions like: Delay, Digest and other conditional steps.
+The template steps are used to organize the different messages in a particular flow. You can add messages for multiple channels and in the upcoming workflow editor release you will be able to configure custom flows that include actions like Delay, Digest and other conditional steps.
 
 ## Messages
 
 A message is particularly tied to a specific channel and will create the content template associated with its channel.
 
-For email channel, you can either use our basic visual editor or a fully custom code with [handlebars variables](https://handlebarsjs.com/guide/).
+For email channels, you can either use our basic visual editor or a fully custom code with [handlebars variables](https://handlebarsjs.com/guide/).
```
Comment (p-fernandez) [inline docs/docs/platform/templates.md:31]: Channel in Novu is any type of notification used: email, sms, etc. So the use here was rightfully singular. Please roll back the change.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Some good additions but other bits that need to be rolled back.
🌟

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**